code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -137,7 +137,7 @@
 					return
 				to_chat(user, span_notice("You start to wire [src]..."))
 				if(do_after(user, 40, target = src))
-					if(coil.get_amount() >= 1 && build_step == 6)
+					if(coil.get_amount() >= 1)
```

Comment:
```suggestion
					if(coil.get_amount() >= 1 &&  build_step = ASSEMBLY_SEVENTH_STEP)
```
this check is here to make sure that two people can't attach cable coils to an ED-209 assembly at the same time to skip past the step that requires a disabler